examples/alerts/new_github_stars.yml

```diff
@@ -0,0 +1,35 @@
+# Alert when there are new GitHub Stars utilizing keepstate
+alert:
+  id: new-github-stars
+  description: Get new GitHub Stars
+  steps:
+    - name: get-github-stars
+      provider:
+        type: github.stars
+        config: " {{ providers.github-pat }}"
```

Comment:
why do I need github PAT for public repo such as Keep? Maybe we can eliminate it? and then the usage can be easier

Reply:
it's not mandatory and can live w/o BUT you are very much limited with API rate limits